lib/test.js

```diff
@@ -108,6 +109,12 @@ Test.prototype.test = function (name, opts, cb) {
 
 Test.prototype.comment = function (msg) {
     var that = this;
+
+    // Previous behavior involved `toString` calls on objects, i.e. emitting `[object Object]`.
```

Comment:
No, I'm suggesting always stringifying the first argument no matter what, and not adding the ability to implicitly log object contents.